lib/reboot_cmds.py

```diff
@@ -58,38 +59,27 @@ def run_command(cmd, cfg: config.UAConfig):
 
 
 def fix_pro_pkg_holds(cfg: config.UAConfig):
-    status_cache = cfg.read_cache("status-cache")
-    if not status_cache:
+    fips = FIPSEntitlement(cfg)
+    fips_status, _ = fips.application_status()
```

Comment:
One concern here is that we would be evaluating the fips status for the current setup. 
If the machine has all the fips package holds in place, but don't have the metapackage for some reason, we might not have the `/proc/sys/crypto/fips_enabled` file and show the service as disabled.

Reply:
I guess that's what I thought this was supposed to be doing. The previous version checked the status cache, which is the status as of the last time `pro status` got executed as root, which feels less accurate.

Or are you saying we want to always remove fips holds even if fips isn't enabled currently?

Maybe, I'm misunderstanding the intent of this section :upside_down_face:

Reply:
@lucasmoura updated!